### ISSUE TITLE
Add second resource to read configuration from

### ISSUE THREAD

**Issue by fortysix2ahead** [tinylog-org/tinylog#75]
Log4J 2 has a nice feature where the log configuration is read from the resource log4j2-test.properties and only if this resource does not exist it reads the configuration from the default resource log4j2.properties. This way it's easily possible to have a "develop/debug" configuration and a "production" configuration in parallel and turn a debug build into a production build by just not shipping the -test.properties file.

Would be nice to have a similar mechanism in tinylog ...

**Comment by pmwmedia** [comment]
It's on the todo list for tinylog 2.1. Already with tinylog 1.x you can have a second properties file for developing and just add `-Dtinylog.configuration=tinylog-test.properties` to your run configuration.

**Comment by fortysix2ahead** [comment]
Yes, that's exactly what I am doing currently!
